frontend/src/app/app.module.ts

```diff
@@ -11,35 +11,35 @@ import { CommonModule } from '@angular/common';
 import { HTTP_INTERCEPTORS, HttpClientModule } from '@angular/common/http';
 import { NgModule } from '@angular/core';
 import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { MatAutocompleteModule } from '@angular/material/autocomplete';
 import { MatBadgeModule } from '@angular/material/badge';
+import { MatButtonModule } from '@angular/material/button';
```

Comment:
URL: http://localhost:4200/project/coffee-machine/model/coffee-machine/metadata
Issue:
- Remove Name of model (Coffee machine), it's already displayed in the form field directory below the title
Screenshot:
![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/81ca036c-7c3b-4890-8cc2-0819f2d51122)